release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -3016,6 +3016,62 @@ This section will continue to be updated over time to provide notes on enhanceme
+
+* Previously, a bug occurred due to insufficient enforcement of a minimum number of services for {ibm-cloud-title} in the `HostedControlPlane` and `HostedCluster` specifications within the code. This issue led to potential data loss or incorrect processing of user-entered data, causing unexpected application behavior. With this release, the problem causing inaccurate data to display in the user interface is corrected, ensuring more reliable and precise information for end users. (link:https://issues.redhat.com/browse/OCPBUGS-54609[*OCPBUGS-54609*])
+
+* Previously, the Secret Janitor was not properly scoped in the `hypershift-operator`, which caused improper secret cleanup. This resulted in token secrets accumulating over time, disrupting the secret management process, while using annotation scoping with two instances of the `hypershift-operator`. With this release, a fix ensures that the secret cleanup continues as expected in a Red Hat OpenShift Kubernetes Service (ROKS) cluster that is managed by the `hypershift-operator`. The large amount of token secrets is eliminated and the proper secret management is maintained. (link:https://issues.redhat.com/browse/OCPBUGS-54498[*OCPBUGS-54498*])
+
+* Previously, a bug occurred due to the incorrect handling of the etcd URL, preventing access to the `Kyverno` service. This resulted in DNS errors during the `kyverno` validation, preventing users on a {product-title} cluster with {hcp} from creating additional test groups. With this release, users can create additional test groups without encountering DNS errors during the `kyverno` validation. (link:https://issues.redhat.com/browse/OCPBUGS-54411[*OCPBUGS-54411*])
+
+* Previously, insufficient permissions led to the persistence of the `disk.csi.azure.com/agent-not-ready=value:NoExecute` taint after creating new {azure-first} disk Container Storage Interface (CSI) driver nodes. With this release, a fix disables the removal of the `not-ready` taint for {azure-short} disk CSI driver nodes, making the scheduler adhere to the `volume-attach-limit`. (link:https://issues.redhat.com/browse/OCPBUGS-54383[*OCPBUGS-54383*])
+
+* Previously, containers that used the SELinux domain of `container_logreader_t` to view container logs on a host at `/var/log` could not access logs in the `/var/log/containers` subdirectory. This was because of a missing symbolic link. With this release, a symbolic link is created for `/var/log/containers` so the containers can access the logs in `/var/log/containers`. (link:https://issues.redhat.com/browse/OCPBUGS-54342[*OCPBUGS-54342*])
```

Comment:
```suggestion
* Previously, containers that used the SELinux domain of `container_logreader_t` to view container logs on a host in the `/var/log` directory could not access logs in the `/var/log/containers` subdirectory. This was because of a missing symbolic link. With this release, a symbolic link is created so the containers can access the logs in the `/var/log/containers` subdirectory. (link:https://issues.redhat.com/browse/OCPBUGS-54342[*OCPBUGS-54342*])
```